src/Thorn_Calculus-Core_Bisimilarities.thy

```diff
@@ -1616,10 +1615,7 @@ next
       B \<triangleright>\<^sup>\<infinity> y. (A \<triangleright>\<^sup>\<infinity> x. \<P> x \<parallel> \<Q> y)
       \<rightarrow>\<^sub>s\<lparr>B \<triangleright> \<star>\<^bsup>n\<^esup> Y\<rparr>
       post_receive n Y (\<lambda>y. A \<triangleright>\<^sup>\<infinity> x. \<P> x \<parallel> \<Q> y) \<parallel> (B \<triangleright>\<^sup>\<infinity> y. (A \<triangleright>\<^sup>\<infinity> x. \<P> x \<parallel> \<Q> y)) \<guillemotleft> suffix n"
-      (is "B \<triangleright>\<^sup>\<infinity> y. ?\<R> y \<rightarrow>\<^sub>s\<lparr>_\<rparr> _")
-      using receiving [where \<P> = "\<lambda>y. ?\<R> y \<parallel> B \<triangleright>\<^sup>\<infinity> y. ?\<R> y"]
-      unfolding post_receive_after_parallel
-      by (subst repeated_receive_proper_def) force
+      by (fact repeated_receive_transition)
```

Comment:
This can be simplified to `using repeated_receive_transition .`.

Reply:
Fixed in 334f4bb7454f9e0671ab3fd3fb31b8e7c37b0a52, please verify.

Reply:
Looks good.